### PR TITLE
adding libfdk-aac

### PR DESCRIPTION
**Pull request by kcc**
Do not merge just yet, I want to see who will be interested in being added to CC

### CODE REVIEW

**Comment by oliverchang** [comment]
Any updates here?

**Comment by kcc** [comment]
Not yet, sorry. I haven't forgotten :)

**Comment by kcc** [comment]
this is ready for review now.

File: projects/libfdk-aac/aacDecoder_Open.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  INT nrOfLayers;
+
+  if (Size != 8) return 0;
+
+  transportType = (TRANSPORT_TYPE)(Data[0] + (Data[1] << 8) + (Data[2] << 16) +
+                                   (Data[3] << 24));
+  nrOfLayers =
+      (UINT)(Data[4] + (Data[5] << 8) + (Data[6] << 16) + (Data[7] << 24));
+
+  aacDecoderInfo = aacDecoder_Open(transportType, nrOfLayers);
```
Comment (nedwill) [inline projects/libfdk-aac/aacDecoder_Open.cpp]: This seems like a fairly shallow target. I suppose opening with malicious (transportType, nrOfLayers) pairs have been buggy in the past?

**Pull request merged**